src/components/views/settings/DevicesPanel.js

```diff
@@ -143,11 +143,22 @@ export default class DevicesPanel extends React.Component {
                     continueKind: "danger",
                 },
             };
-            Modal.createTrackedDialog('Delete Device Dialog', '', InteractiveAuthDialog, {
-                title: _t("Authentication"),
+            const signOutDevs = this.state.devices.filter(dev => this.state.selectedDevices.includes(dev.device_id));
+            const signOutDevsDiv =
+                <div className="mx_DevicesPanel mx_DevicesPanel_signout_devices">
+                    <div className="mx_DevicesPanel_header">
+                        <div className="mx_DevicesPanel_deviceId">{ _t("ID") }</div>
+                        <div className="mx_DevicesPanel_deviceName">{ _t("Public Name") }</div>
+                        <div className="mx_DevicesPanel_deviceLastSeen">{ _t("Last seen") }</div>
+                    </div>
+                    { signOutDevs.map(dev => this._renderDevice(dev, false)) }
+                </div>;
+            Modal.createTrackedDialog('Sign Out Device Dialog', '', InteractiveAuthDialog, {
```

Comment:
historically we've left the dialog tracking state alone, even if we change the name of the buttons. I don't think that's a real concern for us at the moment though.

Reply:
Thanks for the review. I keep the change then.